controller/src/beerocks/bml/internal/bml_internal.h

```diff
@@ -152,6 +152,25 @@ class bml_internal : public beerocks::socket_thread {
     int bml_set_vap_list_credentials(const BML_VAP_INFO *vaps, const uint8_t vaps_num);
     int bml_get_vap_list_credentials(BML_VAP_INFO *vaps, uint8_t &vaps_num);
 
+    //set and get channel scan enable flag
+    int set_dcs_continuous_scan_enable(const std::string &mac, int enable);
+    int get_dcs_continuous_scan_enable(const std::string &mac, int *output_enable);
+
+    //set and get channel scan params
+    int set_dcs_continuous_scan_params(const std::string &mac, int dwell_time, int interval_time,
```

Comment:
Don't use std::string for mac, use sMacAddr.

Reply:
[See](https://github.com/prplfoundation/prplMesh/pull/694/commits/3104d970c25b24adcf45be9e062c5d8b5a68a96e#r369162085)